### ISSUE TITLE
Integrate with react-hook-form

### ISSUE THREAD

**Issue by GabrielGomesBarbosa** [kenshoo/react-multi-select#182]
Hi folks,

How can i use this library with `react-hook-form` correctly? can anyone post some examples?

**Comment by rickyalmeidadev** [comment]
Yes, you must use `Controller` component with its `render` prop. I made a very simple example [here](https://codesandbox.io/s/react-multi-select-integration-with-react-hook-form-wuvon?file=/src/App.js).
